### PR TITLE
Mobile middle section (My Portfolio Project)

### PR DESCRIPTION
**Pull request by DanielAD10**
PR Description

Project Name: My Portfolio
Tech: Built with HTML & CSS
Author: Daniel Adams (adamsdaniel171@gmail.com)

In this PR,

Added work section & about myself section in index.html
Styled work section & about myself section
Added and updated style.css
Updated in README.md

After getting code review,

Updated code in index.html and style.css that need to change

Thank you for reviewing my PR

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -35,6 +35,148 @@ <h1 class="primary-text">Hey there. <br>
         <div class="btn">
             <img src="assets/images/normalbottun.svg" alt="normalbottun">
         </div>
+        <section class="section-container">
+            <div class="text-container">
+                <h1 class="primary-text work-text">My Recent Works</h1>
```
Comment (braincee) [inline index.html]: - [ ] Please I suggest changing the `h1` tag of the "My Recent Works" text to `h2` - `h6` tags to implement the correct flow of HTML semantics for a better user experience. 😄 
Please check this [link](https://developer.mozilla.org/en-US/docs/Glossary/Semantics) for more information. 😄

File: index.html [hunk 2]
```diff
@@ -35,6 +35,148 @@ <h1 class="primary-text">Hey there. <br>
         <div class="btn">
             <img src="assets/images/normalbottun.svg" alt="normalbottun">
         </div>
+        <section class="section-container">
```
Comment (braincee) [inline index.html:38]: - [ ] Please I suggest changing the `section-container` class from `display: flex` property to `display: grid` per the project requirements. 😄

File: index.html [hunk 3]
```diff
@@ -35,6 +35,148 @@ <h1 class="primary-text">Hey there. <br>
...
+            <div class="text-container">
+                <h1 class="primary-text work-text">My Recent Works</h1>
+                <img src="assets/images/indicator.svg" alt="">
+            </div>
+            <div class="card-workflow">
+                <div class="card">
+                    <div class="text-box">
+                        <h1 class="text">Multi-Post Stories Gain+Glory</h1>
```
Comment (braincee) [inline index.html]: - [ ] Please I suggest changing the h1 tag of the "Multi-Post Stories Gain+Glory" text to h2 - h6 tags to implement the correct flow of HTML semantics for a better user experience. Please these changes should be made to each `card` class of the section. 😄
Please check this [link](https://developer.mozilla.org/en-US/docs/Glossary/Semantics) for more information. 😄

File: index.html [hunk 4]
```diff
@@ -35,6 +35,148 @@ <h1 class="primary-text">Hey there. <br>
...
+                        <ul class="list-box">
+                            <li class="list-items">Ruby on Rails</li>
+                            <li class="list-items">CSS</li>
+                            <li class="list-items">JavaScript</li>
+                            <li class="list-items">HTML</li>
+                        </ul>
+                        <button class="btn card-btn">See Project</button>
```
Comment (braincee) [inline index.html:53]: - [ ] Please each `button` should have a hover effect per the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-468&t=DUj2Dpiew61pAJwT-0)

This is a screenshot of the Figma design hover effects. 😄 

Screenshot (285)

File: index.html [hunk 5]
```diff
@@ -35,6 +35,148 @@ <h1 class="primary-text">Hey there. <br>
...
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="section-container">
+            <div class="text-container">
+                <h1 class="primary-text about-me-1">About me</h1>
```
Comment (braincee) [inline index.html]: - [ ]  Please I suggest changing the `h1` tag of the "About Myself" text to `h2 - h6` tags to implement the correct flow of HTML semantics for a better user experience. 😄
Please check this [link](https://developer.mozilla.org/en-US/docs/Glossary/Semantics) for more information. 😄

File: index.html [hunk 6]
```diff
@@ -35,6 +35,148 @@ <h1 class="primary-text">Hey there. <br>
...
+                <p class="third-text about-me-2"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hesitate to contact me.</p>
+                <button class="btn card-btn about-me-3">Get my resume</button>
```
Comment (braincee) [inline index.html:123]: - [ ] Please the `Get my resume` button should have a hover effect per the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-468&t=DUj2Dpiew61pAJwT-0)
This is a screenshot of the Figma design hover effects. 😄

Screenshot (285)

File: index.html [hunk 7]
```diff
@@ -35,6 +35,148 @@ <h1 class="primary-text">Hey there. <br>
...
+            </div>
+            <div class="card-workflow">
+                <div class="card">
+                    <div class="logo">
+                        <img src="assets/images/language.svg" alt="Language logo">
+                    </div>
+                    <div class="box">
+                        <h1 class="text common-text">Language</h1>
```
Comment (braincee) [inline index.html]: - [ ] Please I suggest changing the h1 tag of the "Language" text to h2 - h6 tags to implement the correct flow of HTML semantics for a better user experience. In addition, make same changes to the "Frameworks" and "Skills" text on the `card-workflow` class. 😄 
 Please check this [link](https://developer.mozilla.org/en-US/docs/Glossary/Semantics) for more information. 😄

**Review by braincee — CHANGES_REQUESTED** [review]
Hello @DanielAD10 👋 , I am @braincee your Code Reviewer

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

HIGHLIGHTS
------

 - GitHub flow rightfully used ✅
 - Meaningful title and summary for PR ✅
 - Professional README.md file ✅
 - No linter errors ✅
 - Great implementation of mobile version design. 🤝 

REQUIRED CHANGES ♻️
------
**_Please check the comments under the review._**

OPTIONAL CHANGES
--------
**_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @braincee in your question so I can receive the notification.

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

----

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (wayungi) [reply index.html:53]: - [ OPTIONAL] Good work implementing the hover effect. I suggest that you also implement the `pressed` effect as per the design.

**Review by wayungi — APPROVED** [review]
Hi @DanielAD10 , 

image

Great work, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**Remember: Every finish line 🏁  is the beginning of a new race 🐎** 

## Highlights
- [ ] All linter checks are passing ✔️ 
- [ ] Good commit messages ✔
- [ ]  Correct GitHub flow ✔️ 
- [ ] Descriptive PR body ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**